site/app/controllers/grading/SimpleGraderController.php

```diff
@@ -131,4 +134,80 @@ public function save($action) {
         $this->core->getOutput()->renderJson($response);
         return $response;
     }
+
+    public function csv($action) {
```

Comment:
`csv` is a very generic name that doesn't adequately describe what the function does.